### PR TITLE
push vector config updates

### PR DESCRIPTION
**Pull request by pgvishnuram**
## Summary of changes

push vector config updates 

## Images are updated by this PR

- [ ] alertmanager
- [ ] auth-sidecar
- [ ] awsesproxy
- [ ] blackbox-exporter
- [ ] configmap-reloader
- [ ] curator
- [ ] dind-golang
- [ ] elasticsearch
- [ ] elasticsearch-exporter
- [ ] fluentd
- [ ] git-daemon
- [ ] git-sync
- [ ] grafana
- [ ] keda
- [ ] keda-metrics-apiserver
- [ ] kibana
- [ ] kube-state
- [ ] kubed
- [ ] nats-exporter
- [ ] nats-server
- [ ] nats-streaming
- [ ] nginx
- [ ] nginx-es
- [ ] node-exporter
- [ ] openresty
- [ ] pgbouncer
- [ ] pgbouncer-exporter
- [ ] postgres-exporter
- [ ] postgresql
- [ ] prometheus
- [ ] redis
- [ ] registry
- [ ] statsd-exporter
- [x] vector

## Checklist

Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.

- [ ] version.txt is updated in the changed image(s)

If a security scan fails for an unchanged image...

- [ ] a fix has been applied OR...
- [ ] an [issue]() has been created

If adding a new image:

- [ ] the directory has the same name you intend it to be published as, less a preceding "ap-"
- [ ] the directory includes a file "version.txt", with version matching the underlying software version
- [ ] the file .github/PULL_REQUEST_TEMPLATE.md is updated to include the image in the checklist
- [ ] execute the script .circleci/generate_circleci_config.py, commit changes to .circleci/config.yml

### CODE REVIEW

File: vector/vector.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Vector Sample Config
+# Vector Sample Configuration for defaults
```
Comment (danielhoherd) [inline vector/vector.toml:1]: I think this whole file should be replaced with defaults that actually function. I do not like the idea of having some dummy data showing up in a cluster. It should either be real data, or it should exit.

EG, this snip that was generated using `dasel -r yaml -f foo.toml -w toml` against the `vector-config.yaml` file found in <https://github.com/astronomer/airflow-chart/pull/312/files#diff-06265d7d34eecf75ee03e54e085ff4cf55fd01b13339ec14ba2d9e19cffd7002R16>

```
data_dir = "${SIDECAR_LOGS}"

[sinks]

  [sinks.out]
    compression = "none"
    inputs = ["transform_syslog"]
    mode = "bulk"
    type = "elasticsearch"

[sources]

  [sources.generate_syslog]
    include = ["${SIDECAR_LOGS}/*.log"]
    read_from = "beginning"
    type = "file"

[transforms]

  [transforms.transform_syslog]
    inputs = ["generate_syslog"]
    type = "add_fields"

    [transforms.transform_syslog.fields]
      component = "${COMPONENT:--}"
      release = "${RELEASE:--}"
      workspace = "${WORKSPACE:--}"
```

Comment (pgvishnuram) [reply vector/vector.toml:1]: let me test with this config locally

Comment (danielhoherd) [reply vector/vector.toml:1]: @pgvishnuram any word on this?

Comment (danielhoherd) [reply vector/vector.toml]: This was fixed in another PR.

**Pull request closed without merging**